data/liveinst/locale1-x11-sync.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Sync localed configuration with running X11 system
```

Comment:
Missing `Documentation` which is recommended by Packaging guidelines. So I'm thinking if we want to add `Documentation` field with link to the script installed locally.
https://docs.fedoraproject.org/en-US/packaging-guidelines/Systemd/#unit_files_basic_format_unit_doc

Reply:
Or just point to https://github.com/rhinstaller/localed-x11-sync ?

Reply:
It's not the same code but why not.